components/identifier-generator/front/src/feature/tabs/SelectionTab.tsx

```diff
@@ -28,13 +44,15 @@ const SelectionTab: React.FC<SelectionTabProps> = ({target, conditions, onChange
     }))
   );
 
-  const removeIdentifiers: (conditionsWithId: ConditionsWithIdentifier) => Conditions = conditionsWithId => {
-    return conditionsWithId.map(conditionWithId => {
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      const {id, ...condition} = conditionWithId;
+  const removeIdentifier: (conditionWithId: ConditionWithIdentifier) => Condition = conditionWithId => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const {id, ...condition} = conditionWithId;
+
+    return condition;
+  };
 
-      return condition;
-    });
+  const removeIdentifiers: (conditionsWithId: ConditionsWithIdentifier) => Conditions = conditionsWithId => {
```

Comment:
```suggestion
  const removeIdentifiers = (conditionsWithId) => conditionsWithId.map(removeIdentifier);
```
Would that work simply by being curried? (It would look nicer but I get it if you think it's less understandable)